app/support/notification_sender.rb

```diff
@@ -7,8 +7,17 @@ def initialize(current_facility, order_detail_ids)
     @order_detail_ids = order_detail_ids
   end
 
+  def init_account_ids_to_notify
+    @account_ids_to_notify =
```

Comment:
I tried this earlier, but it's complicated. The tricky part is that if we wait to set `account_ids_to_notify` when it's used, that happens *after* the order details in question are marked as reviewed, meaning we'll get an empty set.